src/baldr/edgeinfo.cc

```diff
@@ -7,6 +7,14 @@ using namespace valhalla::baldr;
 
 namespace {
 
+// should return true for any tags which we should consider "named"
+bool IsNameTag(char ch) {
+  return true;
+  // static const std::unordered_set<TaggedValue> kNameTags = {TaggedValue::kBridge,
+  //                                                           TaggedValue::kTunnel};
+  // return kNameTags.count(static_cast<TaggedValue>(static_cast<uint8_t>(ch))) > 0;
```

Comment:
this looked good to me, why comment it out?

Reply:
oops, sorry, I was experimenting locally and accidentally pushed it
will fix soon